### ISSUE TITLE
Release for Edge browser is not working

### ISSUE THREAD

**Issue by PKief** [material-extensions/material-icons-browser-extension#95]
According to the logs the release of the extension to the edge store is not working properly:

![image](https://github.com/material-extensions/material-icons-browser-extension/assets/12248527/9f41bbbd-118d-496a-af40-dcef27730192)

@shivapoudel On the [edge addons page](https://microsoftedge.microsoft.com/addons/detail/material-icons-for-github/khckkdgomkcjjnpgjmdmbceiddlmiolb) I can see, that the current version is 1.8.13. So it didn't work for 1.8.14 and 1.8.15. Do you know what's the reason for this? Is it something that can be solved easily?

The last successful build was on 10th of May: https://github.com/material-extensions/material-icons-browser-extension/actions/runs/9026752280/job/24804544477 for version 1.8.13.

**Comment by shivapoudel** [comment]
@PKief Any idea when this was published "UPDATE - Material Icons for Web Version 1.8.18". This was happening because it takes few days to approve the extension and during the release it was still in progress.
<img width="1800" alt="Screenshot 2024-07-06 at 9 42 49 AM" src="https://github.com/material-extensions/material-icons-browser-extension/assets/3774827/17031084-5bee-46ce-81fd-77cc6fe4e14c">

**Comment by shivapoudel** [comment]
@PKief Now I have canceled the version 1.8.18 review and manually published the draft one 1.8.15 and it will require a few days to get approved.
<img width="1508" alt="Screenshot 2024-07-06 at 9 48 29 AM" src="https://github.com/material-extensions/material-icons-browser-extension/assets/3774827/3890e43c-9669-4381-b110-6e00b4b394c2">

**Comment by PKief** [comment]
@shivapoudel Thanks a lot for the quick support. Version 1.8.15 should be the correct version to go live at the moment. The version mismatch with version 1.8.18 was due to bringing together the temporary fork with the upstream repository.

**Comment by shivapoudel** [comment]
Again, version 1.8.15 is in the queue for review; another version 1.8.16 is in the draft again. The daily check for the release script may be too fast!
![image](https://github.com/material-extensions/material-icons-browser-extension/assets/3774827/02938ccb-05d3-41d6-b25d-16cea74033be)

**Comment by PKief** [comment]
@shivapoudel Thanks for the update! I just investigated why it tried to publish a new version and it was due to a mistake in the release workflow. It didn't find the installed package version because the installation command was not there. I've added it with https://github.com/material-extensions/material-icons-browser-extension/commit/5a9602e5a33878167e0de90bae56f77855b98029. I think it shouldn't trigger an update everyday anymore, because the version check should compare both versions and not proceed if the versions are the same.

Let's wait for Microsoft to review the versions. Version 1.8.16 is now the latest version, the other browser extensions for chrome and firefox have been updated as well, so let's keep it for now.
